pontoon/base/models/entity.py

```diff
@@ -845,9 +846,13 @@ def for_project_locale(
         if search:
             search_list = utils.get_search_phrases(search)
 
+            q_rejected = (
+                Q()
+            )  # if search_rejected_translations else Q(translation__approved=True)
```

Comment:
That's not a completely accurate condition, because we should also search in suggestions, so:
```suggestion
            )  # if search_rejected_translations else Q(translation__rejected=False)
```